.github/workflows/prbot.yml

```diff
@@ -84,7 +84,14 @@ jobs:
       - uses: actions/checkout@v2
       - uses: ./.github/actions/setup
       - uses: ./.github/actions/e2e-vuejs
-
+  e2e-atomic-stencil-test:
```

Comment:
Would we want to run it also on main branch ?

This will only run on pull request

Reply:
Oh yes, I'll add it to the other workflows!